src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Application {
+    public static void main(String[] args) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        File reportFile = new File("report.csv");
+        try {
+            reportFile.createNewFile();
```

Comment:
There is no need to explicitly create a new file using `createNewFile()`. The `WorkWithFile.getStatistic` method should handle file creation if it's necessary. This will simplify the `main` method and encapsulate the file operations within the `WorkWithFile` class.